3_javascript/javascript1/week4/script4.js

```diff
@@ -0,0 +1,104 @@
+let userName = [];
+const toDoList = [];
+let timer;
+
+function getReply(command) {
+    if (command.includes("Hello my name is")) {
+        const name = command.split("Hello my name is ")[1];
+
+        if (userName.includes(name)) {
+            return "You have already said that!";
+        }
+
+        userName.push(name);
+
+        return `Nice to meet you ${name}`;
+    }
+
+    return "I'm sorry, I don't understand that command.";
+}
+
+if (command === "what is my name") {
```

Comment:
An idea here could be to convert to ```toLowerCase``` https://www.geeksforgeeks.org/javascript-string-tolowercase-method/.

This is just in case the user makes a mistake and writes ```What is my Name``` for example.